query_processor/services/content_router.py

```
@@ -0,0 +1,113 @@
+                'status': result.get('status', 'completed'),
+                'message': 'Topic query processing completed',
+                'query': query_request.original_content,
+                'concepts': result.get('concepts', []),
+                'enhanced_queries': result.get('enhanced_queries', []),
+                'intent_type': result.get('intent_type', ''),
+                'video_urls': result.get('video_urls', []),
+                'total_videos': result.get('total_videos', 0)
+            }
+            
+        except Exception as e:
+            error_msg = f"Failed to route topic request: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)